src/main/java/org/opentripplanner/graph_builder/module/DirectTransferGenerator.java

```diff
@@ -88,6 +89,19 @@ public void buildGraph(
                 );
                 n += 1;
             }
+            if (OTPFeature.FlexRouting.isOn()) {
+                for (StopAtDistance sd : nearbyStopFinder.findNearbyStopsConsideringPatterns(ts0,  true)) {
+                    /* Skip the origin stop, loop transfers are not needed. */
+                    if (sd.stop == ts0.getStop())
+                        continue;
+                    if (sd.stop instanceof Stop)
```

Comment:
Stop to stop transfers are already populated by the `findNearbyStopsConsideringPatterns(ts0, false)`